### PR TITLE
fix(ux): Adjust table column widths and overflow badges by truncating them

### LINKED ISSUE #5230: `/sites` - fix table column size

### LINKED ISSUE #5233: `/groups` - fix column widths

### LINKED ISSUE #5235: `/policies` - adjust column sizes

### LINKED ISSUE #5237: `/identity_providers` - adjust column sizes

### LINKED ISSUE #5244: `/actor` - fix overflow in table cells (ipv6)

### LINKED ISSUE #5245: `/actor` - reduce auth/expires column widths

### LINKED ISSUE #5247: `/groups` - actors table column widths to prevent horizontal scroll

### LINKED ISSUE #5252: `/actors` - last signed in and identifier column sizes smaller

### PR DESCRIPTION
**Pull request by jamilbk**
Fixes #5230 
Fixes #5244
Fixes #5233 
Fixes #5245 
Fixes #5247 
Fixes #5237 
Fixes #5235 
Fixes #5252 

Updates the sidebar to collapse at the `xl` breakpoint, allowing to stay closed for more screen realestate on smaller screens.

Screenshot 2024-06-05 at 1 06 49 PM

### CODE REVIEW

File: elixir/apps/web/lib/web/live/actors/groups.ex [hunk 1]
```diff
@@ -80,7 +80,7 @@ defmodule Web.Actors.EditGroups do
             ordered_by={@order_by_table_id["groups"]}
             metadata={@groups_metadata}
           >
-            <:col :let={group} label="GROUP">
```
Comment (jamilbk) [inline elixir/apps/web/lib/web/live/actors/groups.ex:83]: Uppercasing is handled by Tailwind's `uppercase`

File: elixir/apps/web/lib/web/components/core_components.ex [hunk 2]
```diff
@@ -853,24 +853,17 @@ defmodule Web.CoreComponents do
       >
         <.provider_icon adapter={@identity.provider.adapter} class="h-3.5 w-3.5" />
       </.link>
-      <span class={~w[
-        flex items-center
+      <span class={~w|
```
Comment (bmanifold) [inline elixir/apps/web/lib/web/components/core_components.ex]: Minor nit:  Was there a specific reason for using `|` as the delimiter?  Just wondering for consistency if we should use `[`?

Comment (jamilbk) [reply elixir/apps/web/lib/web/components/core_components.ex]: Oh, I'll change it back. I had added a tailwind class that had the bracket

File: elixir/apps/web/lib/web/components/core_components.ex [hunk 3]
```diff
@@ -853,24 +853,17 @@ defmodule Web.CoreComponents do
...
-      </span>
-      <span :if={not is_nil(@identity.deleted_at)} class="text-sm">
-        (deleted)
-      </span>
-      <span :if={not is_nil(@identity.provider.disabled_at)} class="text-sm">
-        (provider disabled)
-      </span>
-      <span :if={not is_nil(@identity.provider.deleted_at)} class="text-sm">
-        (provider deleted)
+      |}>
```
Comment (jamilbk) [inline elixir/apps/web/lib/web/components/core_components.ex]:
```suggestion
      ]}>
```

File: elixir/apps/web/lib/web/components/core_components.ex (hunk 2 above)
Comment (jamilbk) [inline elixir/apps/web/lib/web/components/core_components.ex]:
```suggestion
      <span class={~w[
```

Comment (bmanifold) [reply elixir/apps/web/lib/web/components/core_components.ex]: Ah, ok, gotcha.  I mean, if that's a possibility, maybe we should use `|` for all of our delimiters 🤷‍♂️ 
Not a high priority though.

**Review by bmanifold — APPROVED** [review]
LGTM

**Pull request merged**